cli/cmd/profile.go

```diff
@@ -35,30 +50,50 @@ func newCmdProfile() *cobra.Command {
 	options := newProfileOptions()
 
 	cmd := &cobra.Command{
-		Use:   "profile [flags] --template (SERVICE)",
+		Use:   "profile [flags] (--template | --open-api file) (SERVICE)",
 		Short: "Output template service profile config for Kubernetes",
 		Long: `Output template service profile config for Kubernetes.
```

Comment:
Maybe it's worth tweaking these short descriptions to not include the word "template" now? Especially for the long description, this help output ready a bit awkwardly to me:

```
$ bin/go-run cli help profile
Output template service profile config for Kubernetes.
		
This outputs a service profile for the given service.
		
If the --template flag is specified, ...
```